src/main.rs

```diff
@@ -58,6 +58,7 @@ const DEFAULT_LOG_LEVEL: LevelFilter = LevelFilter::Info;
 // Command-line argument and option names
 const TOASTFILE_OPTION: &str = "file";
 const CONFIG_FILE_OPTION: &str = "config-file";
+const CONFIG_OUTPUT_DIR_OPTION: &str = "output-dir";
```

Comment:
I would just call this `OUTPUT_DIR_OPTION`.

Since `output_path` appears last in the `Settings` struct below, can we move this constant to be after the others to match that order?